src/store/gameSlice.ts

```diff
@@ -20,6 +21,7 @@ const initialState: initialStateType = {
   winner: '',
   loser: '',
   gameHistory: [],
+  host: null,
```

Comment:
here you can see when I first go to the page, host is just set to false, so maybe the init the key as false? If so then the type would only need to be boolean
<img width="453" alt="Screen Shot 2023-02-03 at 8 59 08 AM" src="https://user-images.githubusercontent.com/79232977/216662650-bef5acb8-e814-4f7a-9fd7-cb8b3103e3bc.png">

Reply:
i think you are mixing up 'host' and 'isHost' ?